articles/logout.md

```diff
@@ -34,7 +34,7 @@ To provide the list of URLs that the user may be redirected to after logging out
 When providing the URL list, you may:
 
 - Specify multiple, valid comma-separated URLs;
-- Use `*` as a wildcard for subdomains (e.g. `*.example.com`).
+- Use `*` as a wildcard for subdomains (e.g. `http://*.example.com`).
```

Comment:
Because I've had two customers put `*.whatever.com` and open tickets because they couldn't figure out why it wasn't working. One of which referenced this doc as the reason why he entered it that way.